app/routers/api/files.py

```
@@ -853,7 +859,9 @@ def relocate_file(inventory_id: int, request: FileRelocateRequest, db: Session =
     except ValueError as e:
         inventory_entry.status = FileStatus.ACTIVE
         db.commit()
-        raise HTTPException(status_code=status.HTTP_409_CONFLICT, detail=str(e)) from e
+        logger.error(f"Failed to create relocation task: {e}")
```

Ruff (0.15.7)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)